spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright (c) 2021 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[release_notes_11]]
+== Release Notes for MicroProfile Reactive Messaging 1.1
+
+A full list of changes delivered in the 1.1 release can be found at link:https://github.com/eclipse/microprofile-reactive-messaging/milestone/1?closed=1[MicroProfile Reactive Messaging 1.1 Milestone].
+
+=== API/SPI Changes
+- Add support for unmanaged stream injection using `@Inject @Channel(...)` (link:https://github.com/eclipse/microprofile-reactive-messaging/issues/3[#3])
+- Add support for emitters allowing emitting messages from _imperative_ code (link:https://github.com/eclipse/microprofile-reactive-messaging/issues/70[#70])
+- Add support for metrics (link:https://github.com/eclipse/microprofile-reactive-messaging/issues/31[#31])
```

Comment:
I think this should move to the section of Functional Changes as there is no API/SPI update/addition.

Reply:
Done.